### DISCUSSION TITLE
Ability to duplicate tables

### DISCUSSION THREAD

**Discussion by Grayda** [Budibase/budibase#11019] [Ideas]
I'm building an app, and many of the tables I'm adding share some common fields (e.g. "Title", or "Address" or whatever). It'd be nice if I could click on the three dots menu next to a table and pick "Duplicate", then rename it, so I can rapidly add tables that are all rather similar.

I guess I could export the app and edit the file manually, but if you can duplicate components in the designer, it feels like a natural fit to be able to duplicate entire table structures (and possibly the data too?). 

Admittedly I only need to do this for a total of three tables, but I'm sure someone else would appreciate this feature if they're in a situation like me.

**Comment by ConorWebb96** [comment]
Hey @Grayda,

I'd recommend creating a feature request if you think its a feature that should be added to the core product. I thought this did exist, but that could have been before the new grid data section was added. You can choose the feature request option here; https://github.com/Budibase/budibase/issues/new/choose
